### PR TITLE
style: wrap comments

### PR DESCRIPTION
**Pull request by domodwyer**
Adds a rustfmt [config file](https://github.com/influxdata/influxdb_iox/blob/dom/rustfmt-wrapping/rustfmt.toml) and then wraps all the overly long comments.

There's no logic changes - I added the config, ran `cargo fmt` and committed the lot 👍

### CODE REVIEW

File: rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1 @@
+wrap_comments = true
```
Comment (alamb) [inline rustfmt.toml:1]: 🎉

**Review by alamb — APPROVED** [review]
looks good to me, I vote :shipit:

**Review by pauldix — COMMENTED** [review]
hah, I see a number of tables and monodraws got mangled. We'll have to work on that later ;)

**Comment by domodwyer** [comment]
Ah I had scrolled through the whole thing and didn't see any!

I've created #561 for some demangling - diagrams are too important to mangle!

**Pull request merged**